src/configuration/initialResources.json

```diff
@@ -0,0 +1,57 @@
+{
+  "tokens": [
+    {
+      "tokenName": "Random NFT Token",
+      "tokenSymbol": "RNFT",
+      "decimals": 0,
+      "maxSupply": 1000,
+      "supplyKey": {
+        "value": "0xbc6340d8b24bc70cd8826e43a8cce477756a7c0456a438c6cafab1fdacc0e647",
+        "type": "ECDSA"
+      },
+      "mints": [
+        {
+          "CID": "QmNPCiNA3Dsu3K5FxDPMG5Q3fZRwVTg14EXA92uqEeSRXn"
+        },
+        {
+          "CID": "QmZ4dgAgt8owvnULxnKxNe8YqpavtVCXmc1Lt2XajFpJs9"
+        }
+      ],
+      "tokenMemo": "Random NFT token",
+      "tokenType": "NON_FUNGIBLE_UNIQUE",
+      "supplyType": "FINITE"
+    },
+    {
+      "tokenName": "Random FT Token",
+      "tokenSymbol": "RFT",
+      "decimals": 2,
+      "initialSupply": 1000000,
+      "tokenMemo": "Random FT token",
+      "tokenType": "FUNGIBLE_COMMON",
+      "supplyType": "INFINITE"
+    }
+  ],
+  "accounts": [
+    {
+      "balance": 1000,
+      "privateKey": {
```

Comment:
This change looks good and will enhance the user experience! I have a small proposal - 
The user may not want to generate an account/private key beforehand, but just tell us I want 10 account with certain balance and associated tokens. Thus, we may have the privateKey optional, so if its not there we will create the private key and account ourselves